jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -208,6 +229,37 @@ private void authenticatePush() {
                     .call());
   }
 
+  private void saveDocker() {
+    Optional<DockerClient> dockerClient =
+        buildConfiguration.getBaseImageConfiguration().getDockerClient();
+    Verify.verify(dockerClient.isPresent());
+
+    results.tarPath =
+        executorService.submit(
+            () -> new SaveDockerStep(buildConfiguration, dockerClient.get()).call());
```

Comment:
In this case where `SaveDockerStep` doesn't depend on any future value, no need to wrap it in an additional lambda.

`.submit(new SaveDockerStep(buildConfiguration, dockerClient.get()))`